src/offset/local/mod.rs

```diff
@@ -51,7 +51,7 @@ mod tz_info;
 /// let dt: DateTime<Local> = Local::now();
 /// let dt: DateTime<Local> = Local.timestamp(0, 0);
 /// ```
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
```

Comment:
I added this so that we could have, IMHO, a better example for the `and_local_timezone` function. However, I can understand if this might be controversial. If we don't want to do this, I can change up the example in `and_local_timezone` to be something different.

Reply:
We could also use 

```rust
assert!(TypeId::of::<DateTime<Local>>() == dt.type_id());
```

Which I think would be preferable here as it is hard to interpret `PartialEq` and `Eq` on `Local`.

Reply:
I'm not sure that would be good given that the purpose of the example (as far as I can tell) is to demonstrate that the created `DateTime` does indeed have the expected timezone.

If we don't want to add `Eq` and `PartialEq` to `Local`, then I think the next best thing is to create our own timezone using `FixedOffset` and use that. I'll put up a change trying that so we can see what it looks like.